e2e-tests/tests/test_anonymizer.py

```diff
@@ -109,7 +109,7 @@ def test_given_anonymizers_called_then_expected_anonymizers_list_returned():
     response_status, response_content = anonymizers()
 
     expected_response = """
-        ["hash", "mask", "redact", "replace", "encrypt"]
+        ["hash", "mask", "redact", "replace", "encrypt", "custom"]
```

Comment:
Won't it be misleading to get 'custom' in the REST API, but don't support it as an operation in the REST API?

Reply:
get_anonymizers is a package API, so not only used by the REST API. We can potentially just block it in the REST API itself. WIP

Reply:
request is blocked